yarn.lock

```diff
@@ -1725,10 +1725,6 @@ cipher-base@^1.0.0, cipher-base@^1.0.1, cipher-base@^1.0.3:
   dependencies:
     inherits "^2.0.1"
 
-circular-dependency-plugin@^3.0.0:
```

Comment:
@philippotto This PR never added the circular-dependency-plugin on `master`/ `react-controllers-review-base`. I only enabled it on my test branch and removed it before merging. I just doubled checked `yarn.lock` and `webpack.config.json` on `master`/ `react-controllers-review-base` and both don't include plugin. Perhaps Jenkins cache?